dbms/src/Storages/Transaction/KVStore.h

```diff
@@ -228,6 +237,10 @@ class KVStore final : private boost::noncopyable
         UInt64 compact_term,
         bool is_background,
         bool lock_held = true);
+
+    RaftLogEagerGcTasks::Hints getRaftLogGcHints();
+    void applyRaftLogTaskRes(const RaftLogGcTasksRes & res) const;
```

Comment:
It should be `applyRaftLogGcTaskRes`, there are other tasks related to raft, not in this module though.